README.md

```diff
@@ -157,23 +155,29 @@ Search parameters
 #### Get info on user
 
 ```sh
-curl -X 'GET' \
-  'https://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66' \
-  -H 'accept: application/ld+json'
+curl "http://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66"
 ```
 
 Here `ffdb7559-2ad3-4662-9fd4-d69849939b66` should be a `bruger` identifier.
 
 #### Get funktioner
 
 ```sh
-curl -X 'GET' \
-  'https://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66/funktioner' \
-  -H 'accept: application/ld+json'
+curl "http://$(docker compose port nginx 8080)/api/v1/bruger/ffdb7559-2ad3-4662-9fd4-d69849939b66/funktioner"
 ```
 
 Here `ffdb7559-2ad3-4662-9fd4-d69849939b66` should be a `bruger` identifier.
 
+### Calling from another docker container
+
+The `nginx` container has been overridden to have the container name
+`organisation_api`. Calls from other projects already having a `nginx`
+container should use `organisation_api` to make calls to the API, i.e.
```

Comment:
Tell which network to use (in the other docker compose setup) to make the `organisation_api` service available.

Reply:
Added description explaining which network to use and how to do it.